packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -275,8 +284,9 @@ Will double processing times. Use only for debugging purposes.",
     group: "chain",
   },
 
-  "chain.archiveBlobEpochs": {
-    description: "Number of epochs to retain finalized blobs (minimum of MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS)",
+  "chain.archiveDataEpochs": {
```

Comment:
I think we have a way to add aliases?
we shouldn't break by removing the old cli flag
we should add an alias and a TODO to delete on 2.0